apps/labs/posts/array-libraries-interoperability.md

```diff
@@ -0,0 +1,598 @@
+---
+title: 'Array Libraries Interoperability'
+published: October 13, 2021
+author: anirudh-dagar
+description: "Let's dive into the technicalities involved in array libraries interoperability and understand the protocols making this a reality."
+category: [Array API]
+featuredImage:
+  src: /posts/array-libraries-interoperability/feature.png
+  alt: 'A pro and cons list comparing two protocols, uarray and the Array API.'
+hero:
+  imageSrc: /posts/array-libraries-interoperability/hero.png
+  imageAlt:
+    'Meme of Master Splinter leading the baby turtles from TMNT. Splinter
+    represents NumPy, and the turtles represent TensorFlow, CuPy, PyTorch and JAX.'
```

Comment:
@gabalafou I just noticed that the featured image for the blog is this `feature.png`, which was not present initially. Instead, can we have the `ninja_arrays.png` as the feature image?

Sorry for making the requests one by one. I should have caught that before.

Reply:
That seems like a reasonable request to me, but I didn't make that change or the feature.png image. There may have been some reason for it. Do you know, @MarsBarLee?

Reply:
Thanks @MarsBarLee! Yeah I like the image too, hence the request.😛